bokeh/plotting.py

```diff
@@ -410,13 +410,13 @@ def output_file(filename, title="Bokeh Plot", autosave=False, mode="inline", roo
         print("Session output file '%s' already exists, will be overwritten." % filename)
 
 
-def show(obj=None, browser=None, new="tab", url=None):
-    """ 'shows' a plot object or the current plot, by auto-raising the window or tab
-    displaying the current plot (for file/server output modes) or displaying
-    it in an output cell (IPython notebook).
+def show(obj, browser=None, new="tab", url=None):
+    """ 'shows' a plot object, by auto-raising the window or tab displaying the
+    current plot (for file/server output modes) or displaying it in an output
+    cell (IPython/Jupyter notebook).
 
     Args:
-        obj (Widget/Plot object, optional): it accepts a plot object and just shows it.
+        obj (Widget/Plot object): requires a plot object and just shows it.
```

Comment:
A predicate in the english usage is a verb phrase. In your example I would call "plot object" the object, or if being specific, the direct object. All I really meant is that parameter short descriptions should not contain active verb phrases. 

Another way to look at it is that like like to imagine an implicit "IS" between the parameter and the description:
```
obj: IS a plot object to display
```
and that works. In contrast to the original:
```
obj: IS requires a plot object and just shows it.
```
does not make sense. So maybe that is a shorter/better way of stating it: All argument descriptions should read correctly with an implicit "IS".